### ISSUE TITLE
Crash when I:orePileChance=0

### ISSUE THREAD

**Issue by OrdinatorStouff** [elucent/GravelOres#11]
1.12
gravelores-1.7b.jar

When setting I:orePileChance=0 and creating a new world, client crashes. Console log says something about "value must be positive." 

If you need more details let me know

**Comment by OrdinatorStouff** [comment]
[crash-2017-08-06_18.33.21-server.txt](https://github.com/elucent/GravelOres/files/1203662/crash-2017-08-06_18.33.21-server.txt)

**Comment by KnightMiner** [comment]
Ya, don't do that. I really need to make a release where I prevent all the possibly of crashes from the config.

I assume you were trying to disable oregen while still having the ores? Right now its supported to just backlist all the ores, but I'll add a more proper way to do it in the future.

**Comment by OrdinatorStouff** [comment]
Okay that's fair and for now I just turned everything down to 1 but in my defense it's specifically instructed in the config comments to set to 0 to disable:

```
    # Chance per chunk that an ore pile will spawn. Set this to zero to disable generation. [range: 0 ~ 32767, default: 20]
    I:orePileChance=1
```

**Comment by KnightMiner** [comment]
Huh, must be an oversight. The code never had a specific case for it being 0, so its on my TODO list to fix.

**Comment by KnightMiner** [comment]
Fixed in version 1.8, which is now on CurseForge
